notebook.ipynb

```diff
@@ -94,7 +94,7 @@
   },
```

Comment:
<p>This graph is empty, might be because of my computer</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rawan-a-d/Air-Quality-Prediction/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='75'/>